examples/notebooks/mnist.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
Would you mind clearing the notebook errors?

Does the notebook run correctly now?